snap/snapcraft.yaml

```diff
@@ -486,6 +490,14 @@ apps:
       install-mode: disable    
     environment:
       KuiperBaseKey: $SNAP_DATA/kuiper
+      EDGEX__DEFAULT__PORT: 6379
+      EDGEX__DEFAULT__PROTOCOL: redis
+      EDGEX__DEFAULT__SERVER: edgex-redis
```

Comment:
```suggestion
      EDGEX__DEFAULT__SERVER: localhost
```

Reply:
Nice catch (updated) that said, aren't these overrides just going to get overridden by the security-secretstore-setup message bus code?